### DISCUSSION TITLE
为什么是 user 不是 organization

### DISCUSSION THREAD

**Discussion by ouuan** [thu-stat-wiki/thu-stat-wiki.github.io#3] [General]
想问一下使用 user 而不是 org 是有什么原因吗 🤔

**Comment by V1ncent19** [comment]
@ouuan 因为（紧急 google 了一下才有能力回答此问题的） v1ncent19 是笨蛋，完全没有 github 项目合作经验，也不知道还有 organization 这个功能，所以暂时傻瓜式地用 collaborator 进行维护了（悲）。求教这两种会有什么显著的区别吗？

**Reply by ouuan** [reply]
感觉没有什么非常显著的区别，但是：

- 看上去比较奇怪（
- 我不太清楚 collaborator 的权限是怎样的，org 的话可以分 admin / maintain / write 之类不同等级的权限
- collaborator 只能一个一个仓库给，如果有多个仓库会比较麻烦（感觉倒是问题不大）
- 创建 user 比创建 org 麻烦（只不过已经创建了）
- GitHub 的 tos 其实是[禁止创建小号](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#3-account-requirements)的，只不过这种实质上是 org 的号应该不会被 ban；而且也说了禁止多人共享一个帐号
- 切帐号会麻烦

**Reply by V1ncent19** [reply]
害，我刚刚就忘记切号了（其实是浏览器双开， edge 上官号 chrome 上自己的 hhhhh ）。那看起来好像确实还是应该变成 org 模式，虽然其实应该网站应该也不会规模大到需要很复杂的运营功能。
& 感谢提供 [账户转换指引](https://github.com/thu-stat-wiki/thu-stat-wiki.github.io/discussions/3#discussioncomment-5424730)  ！会快速研究一下方法和适用性。

**Comment by ouuan** [comment]
FYI: [Converting a user into an organization - GitHub Docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/converting-a-user-into-an-organization)

**Reply by V1ncent19** [reply]
success ，已经成功变成 org 了，会研究一下组织模式，谢谢捏！
